source/non-pattern-templates/for-head.mustache

```diff
@@ -0,0 +1,7 @@
+{{! These are the fonts specified in the the css's @font-face rules. }}
+
+<link rel="preload" href="{{#assetRewrite}}/fonts/249228f0-61ac-40cc-a5a5-5609c9816e3f.woff2{{/assetRewrite}}" as="font">
+
+<link rel="preload" href="{{#assetRewrite}}/fonts/627fbb5a-3bae-4cd9-b617-2f923e29d55e.woff2{{/assetRewrite}}" as="font">
```

Comment:
Include `type="font/woff2"`?

Reply:
Yes. It'll need `crossorigin="anonymous"`, too.